op-devstack/sysgo/sequencer.go

```diff
@@ -148,7 +148,7 @@ func WithTestSequencer(testSequencerID stack.TestSequencerID, l2CLID stack.L2CLN
 						Committer: committerID,
 						Publisher: publisherID,
 
-						SequencerConfDepth:  3,
+						SequencerConfDepth:  2,
```

Comment:
I guess this has changed to match the initialization value for `l2_cl.go`'s `SequencerConfDepth`? If possible, we may expose the L2 CL's driver config, and use that to initialize sequencer config here.
Also i noticed that changing this file name to `test_sequencer.go` may bring more consistency.

Reply:
AFAIK this is not exposed yet, but we may do this in a follow up PR